web/src/components/storage/BootSelection.jsx

```diff
@@ -88,7 +87,7 @@ export default function BootSelectionDialog() {
 
       setState({
         load: true,
-        bootDevice: availableDevices.find(d => d.name === bootDevice),
+        bootDevice: availableDevices.find(d => d.name === bootDevice) || availableDevices[0],
```

Comment:
I would like @joseivanlopez to take a look to this part. I understood him there is something wrong beyond that simple fallback.

Reply:
```suggestion
        bootDevice: availableDevices.find(d => d.name === bootDevice) || defaultBootDevice|| availableDevices[0],
```

Reply:
Anyway, I have to do a fix in the rust code. Currently, `defaultBootDevice` is always `undefined`.

Reply:
Sure, if you find a better fix for this in time I'm more than happy to change the introduced hotfix. Think that the BootSelection(Page) component will be re-visited in next iterations, among other components. In any case, I (manually) tested that the patch makes the selection work as expected.